### ISSUE TITLE
Redstone toggles for pipes

### ISSUE THREAD

**Issue by HexBugOrion** [AztechMC/Modern-Industrialization#450]
A block that can cutoff item and fluid pipes with a redstone signal

**Comment by AntonKV-TG** [comment]
Yes, it would be useful to have redstone interaction settings in the pipe interface, like it is done in Ender IO
